nautobot/ipam/filters.py

```diff
@@ -230,9 +252,9 @@ class PrefixFilterSet(
         method="search_contains",
         label="Prefixes which contain this prefix or IP",
     )
-    mask_length = django_filters.NumberFilter(field_name="prefix", lookup_expr="net_mask_length")
-    mask_length__gte = django_filters.NumberFilter(field_name="prefix", lookup_expr="net_mask_length__gte")
-    mask_length__lte = django_filters.NumberFilter(field_name="prefix", lookup_expr="net_mask_length__lte")
+    mask_length = django_filters.NumberFilter(label="mask_length", method="filter_prefix_length_eq")
+    mask_length__gte = django_filters.NumberFilter(label="mask_length", method="filter_prefix_length_eq")
```

Comment:
This looks suspect to me - shouldn't this be `"mask_length__gte"` and `"filter_prefix_length_gte"`?

Reply:
Good catch, I think this was a rebase conflict resolution artifact